specification/cognitiveservices/OpenAI.Inference/models/completions/azure_chat_extensions.tsp

```diff
@@ -1,7 +1,11 @@
 import "@typespec/versioning";
-
 import "./chat_completions.tsp";
-import "./azure_cognitive_search_options.tsp";
+import "./chat_messages.tsp";
+import "./extensions_onyourdata_azure_cognitive_search_options.tsp";
+import "./extensions_onyourdata_azure_machinelearning_index_options.tsp";
+import "./extensions_onyourdata_azure_cosmosdb_options.tsp";
+import "./extensions_onyourdata_elasticsearch_options.tsp";
+import "./extensions_onyourdata_pinecone_options.tsp";
```

Comment:
It looks like the additional data source options got removed for the 2023-12-01-preview swagger that merged to the public repo. Should we leave them out of the typespec as well?

Reply:
I believe they're currently *missing* from the swagger file. We believe they'll still end up implicitly supported since routing will succeed, but we need to follow up with the OYD team to ensure we get the right consistency and APIM support.